src/utils.cpp

```diff
@@ -477,21 +478,54 @@ std::string exec_system(const char* cmd)
   return result;
 }
 
-std::vector<std::string> resolve_binary_path(const std::string& cmd)
+std::vector<std::string> expand_wildcard_path(const std::string& path)
```

Comment:
This and `expand_wildcard_paths` are only used in this file. Can you put them in the anonymous namespace at the top of the file?